### PR TITLE
Multi category

### PR DESCRIPTION
**Pull request by Shamsco**
Refactoring to make code cleaner.
Created Multi-Category visualization for Force-Directed Graph

### CODE REVIEW

File: src/app/arc/arc.component.ts [hunk 1]
```diff
@@ -1,560 +1,654 @@
...
+    { id: 'MotherPlutarch', group: [9] },
+    { id: 'Gueulemer', group: [4] },
+    { id: 'Babet', group: [4] },
+    { id: 'Claquesous', group: [4] },
+    { id: 'Montparnasse', group: [4] },
+    { id: 'Toussaint', group: [5] },
+    { id: 'Child1', group: [10] },
+    { id: 'Child2', group: [10] },
+    { id: 'Brujon', group: [4] },
+    { id: 'Mme.Hucheloup', group: [8] },
```
Comment (mshinoy) [inline src/app/arc/arc.component.ts]: The data should reside in another file. Is there any requirement that the data needs to be in this component? 

Create another file where you store the links and other data, Then use that in both the arc diagram and the force directed graph.

File: src/app/disjointed-graph/disjointed-graph.component.ts [hunk 2]
```diff
@@ -1,495 +1,639 @@
...
+
+interface link extends d3.SimulationLinkDatum<node> {
+  source: string;
+  target: string;
+  value: number;
+}
+interface pieData {
+  color: string;
+  value: number;
+}
```
Comment (mshinoy) [inline src/app/disjointed-graph/disjointed-graph.component.ts]: Can move this interface to another file too. Then import it here and in the arc graph component

File: src/app/arc/arc.component.ts [hunk 3]
```diff
@@ -1,560 +1,654 @@
...
-    {"source": "Brujon", "target": "Claquesous", "value": 1},
-    {"source": "Brujon", "target": "Montparnasse", "value": 1},
-    {"source": "Mme.Hucheloup", "target": "Bossuet", "value": 1},
-    {"source": "Mme.Hucheloup", "target": "Joly", "value": 1},
-    {"source": "Mme.Hucheloup", "target": "Grantaire", "value": 1},
-    {"source": "Mme.Hucheloup", "target": "Bahorel", "value": 1},
-    {"source": "Mme.Hucheloup", "target": "Courfeyrac", "value": 1},
-    {"source": "Mme.Hucheloup", "target": "Gavroche", "value": 1},
-    {"source": "Mme.Hucheloup", "target": "Enjolras", "value": 1}
+    { source: 'Napoleon', target: 'Myriel', value: 1 },
```
Comment (WafaWaheedaS) [inline src/app/arc/arc.component.ts]: Move the json data to a separate file.

**Review by WafaWaheedaS — APPROVED** [review]

**Comment by Shamsco** [comment]
-Multi-group on Force graph
-Multi-group on Arc graph
-Show groups + Show Links on Force Graph
-Sort by group or name in Arc graph

**Pull request merged**